CONTRIBUTING.md

```diff
@@ -0,0 +1,152 @@
+# CONTRIBUTING
+
+- [Contributor Guide](#contributor-guide)
+  - [Prerequisites](#prerequisites)
+    - [Create a GitHub account](#create-a-github-account)
+    - [Sign the CLA](#sign-the-cla)
+    - [Code of Conduct](#code-of-conduct)
+    - [Setting up your development environment](#setting-up-your-development-environment)
+  - [Development](#development)
+  - [Testing](#testing)
+    - [Pre-submits and Unit Tests](#pre-submits-and-unit-tests)
+    - [End to End (e2e) Tests](#end-to-end-e2e-tests)
+    - [Linters](#linters)
+  - [Community](#community)
+    - [Communication](#communication)
+
+## Contributor Guide
+
+This guide will help you understand the overall organization of the
+[slsa-github-generator](https://github.com/slsa-framework/slsa-github-generator)
+project, and direct you to the best places to get started contributing. You'll
+be able to pick up issues, write code to fix them, and get your work reviewed
+and merged.
+
+This document is the single source of truth for how to contribute to the code
+base. Feel free to browse the [open issues] and file new ones, all feedback
```

Comment:
```suggestion
base. Feel free to browse the [open issues](https://github.com/slsa-framework/slsa-github-generator/issues) and file new ones, all feedback
```

Reply:
This used the link defined at the bottom. I copied this from the Kubernetes contributor guide, in which this was referring to issues related to the contributing guide itself. I think I want to reword this section a bit.